src/apic-extension/azext_apic_extension/aaz/latest/apic/service/_create.py

```diff
@@ -192,7 +193,7 @@ def content(self):
             _content_value, _builder = self.new_content_builder(
                 self.ctx.args,
                 typ=AAZObjectType,
-                typ_kwargs={"flags": {"client_flatten": True}}
+                typ_kwargs={"flags": {"required": True, "client_flatten": True}}
```

Comment:
The `resource` parameter is marked as required in the spec, and causes this change.